scripts/check-versions.js

```diff
@@ -34,16 +34,19 @@ function parseVersion(raw) {
     return raw;
   }
 
-  const match = /^v?(\d+)\.(\d+)\.(\d+)(?:-(.+))?$/.exec(raw.trim());
```

Comment:
split out the parsing due to [unicorn/no-unsafe-regex](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-unsafe-regex.md) calling out the `(?:-(.+))?` bit